pisek/compile.py

```diff
@@ -72,12 +72,28 @@ def valid_shebang(filepath: str) -> bool:
         return True
 
 
+def manager_flags(dir, manager, extension):
```

Comment:
Asi by mohlo mít v názvu něco s C/C++, když je to pro ně specifické. Nebo být statická metoda.